File: script/php/asset.php
<?php
class Asset {
    public $ticker = null;
    public $evaluation = null;
    public $volume = null;
    public $market_cap = null;
    public $beta = null;

    function __construct() {
        $this->ticker = "INVALID";
        $this->evaluation = 0;
        $this->volume = 0;
        $this->market_cap = 0;
        $this->beta = 0;
    }
}
?>

File: app/models/asset.rb
# == Schema Information
#
# Table name: assets
#
#  id         :integer         not null, primary key
#  ticker     :string(255)
#  evaluation :float
#  volume     :integer
#  market_cap :integer
#  beta       :float
#  date       :date
#

class Asset < ActiveRecord::Base
	default_value_for :day do
    	Date.today
  	end
end
